### PR TITLE
Improved the custom_dock plugin by making the button work

### PR DESCRIPTION
**Pull request by rhulha**
I think supplying a custom_dock plugin demo where one of the buttons actually does something is more informative and helpful for beginners like me.

Changes:
Added a simple little script for the custom_dock button.
Implemented the button pressed event.
The button press now executes: print("Plugin Button Pressed!")

### CODE REVIEW

**Comment by bojidar-bg** [comment]
I think you should remove the default comments in the script.

**Comment by rhulha** [comment]
> I think you should remove the default comments in the script.

I think they are helpful to beginners.

**Comment by rhulha** [comment]
> If this were trying to demonstrate Godot's scripting language as a whole, then I would agree. But this isn't serving that purpose.
Fair enough.
> 
> Also, you should save that scene as a `.tscn` file for the purposes of this repo.
I did not change the extension of the scene.
I just changed the already existing scene file.

File: plugins/custom_dock/button.gd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+func _ready():
+	# Called when the node is added to the scene for the first time.
+	# Initialization here
+	pass
+
+#func _process(delta):
+#	# Called every frame. Delta is time since last frame.
+#	# Update game logic here.
+#	pass
```
Comment (aaronfranke) [inline plugins/custom_dock/button.gd]: Remove all empty functions and default comments.

**Pull request closed without merging**